src/google/protobuf/map.h

```diff
@@ -224,8 +224,8 @@ class UntypedMapIterator {
 };
 
 // These properties are depended upon by Rust FFI.
-static_assert(std::is_trivial<UntypedMapIterator>::value,
-              "UntypedMapIterator must be a trivial type.");
+static_assert(std::is_trivially_default_constructible<UntypedMapIterator>::value,
```

Comment:
Here since we already had quite other checks written already, I added the only "missing" one: trivial default construction.